index.html

```diff
@@ -2,21 +2,25 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
     <script src="https://kit.fontawesome.com/d52fa491d2.js" crossorigin="anonymous"></script>
+    <title>Document</title>
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
+            <li><i class="fa-solid fa-bars"></i></li>   
+        </ul>
     </header>
     <section class="Headline"> 
         <h1>I’m kelvin Glad to see you!</h1>
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ]  Please make sure to use the right font family, font size, color  and font weight  in order to match The Figma design 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/177769412-9ec3dff3-e141-4b2d-9f8b-612a9bd715eb.png)
- Figma
![image](https://user-images.githubusercontent.com/61790587/177769469-54078a8a-df47-4d6e-9279-3f043826380f.png)

Reply:
> * [ ]  As required by the previous reviewer, pease make sure to use the right font size, color  and font weight  in order to match The Figma design
> * Yours
>   ![image](https://user-images.githubusercontent.com/61790587/177769412-9ec3dff3-e141-4b2d-9f8b-612a9bd715eb.png)
> * Figma
>   ![image](https://user-images.githubusercontent.com/61790587/177769469-54078a8a-df47-4d6e-9279-3f043826380f.png)